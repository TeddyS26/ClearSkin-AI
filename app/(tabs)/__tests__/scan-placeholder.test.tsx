import React from "react";
import { render } from "@testing-library/react-native";
import ScanPlaceholder from "../scan-placeholder";

describe("ScanPlaceholder", () => {
  it("should render empty view", () => {
    const { toJSON } = render(<ScanPlaceholder />);
    expect(toJSON()).toBeTruthy();
  });

  it("should match snapshot", () => {
    const { toJSON } = render(<ScanPlaceholder />);
    expect(toJSON()).toMatchSnapshot();
  });

  it("should export a function", () => {
    expect(typeof ScanPlaceholder).toBe("function");
  });
});

