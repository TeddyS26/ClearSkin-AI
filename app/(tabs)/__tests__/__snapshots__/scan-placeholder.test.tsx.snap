// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScanPlaceholder should match snapshot 1`] = `
<View
  className="flex-1 bg-white"
/>
`;
